pecan/decorators.py

```diff
@@ -32,7 +32,9 @@ def expose(template=None,
     access via HTTP, and to configure that access.
 
     :param template: The path to a template, relative to the base template
-                     directory.
+                     directory.  Can also be passed a string representing
+                     a special or custom renderer, such as ``'json'`` for
```

Comment:
Updated with a bit more clarity, though some of this *is* answered in the link specified in this docstring:

https://pecan.readthedocs.org/en/latest/templates.html#the-json-renderer